pkgs/development/ocaml-modules/mirage-block-unix/default.nix

```diff
@@ -15,7 +15,8 @@ buildDunePackage rec {
 
   minimumOCamlVersion = "4.06";
 
-  propagatedBuildInputs = [ cstruct-lwt logs mirage-block rresult uri ];
+  propagatedBuildInputs =
+    [ cstruct-lwt io-page-unix logs mirage-block rresult uri ];
```

Comment:
I don’t understand this change: the `mirage-block-unix` library does not use `io-page-unix` in any way, does it?

Reply:
Its listed as a dependency, but somehow not used during the checkPhase: https://opam.ocaml.org/packages/mirage-block-unix/

Reply:
Does this mean that there is a mistake in the page you’ve linked?

Reply:
Well it is listed in the opam and dune metadata of mirage-block-unix.